src/api/public/apidocs-new/OBS-v2.10.50.yaml

```
@@ -18,6 +18,7 @@ tags:
   - name: Published Binaries
   - name: Requests
   - name: Sources - Projects
+  - name: Search
```

```suggestion
  - name: Search
  - name: Sources - Projects
```
Just to keep items sorted.